actions/transactions/new.go

```diff
@@ -61,3 +61,19 @@ func (a *Action) newTransaction(c *gin.Context) {
 	contract := mappings.MapToDraftTransactionContract(transaction)
 	c.JSON(http.StatusCreated, contract)
 }
+
+// newTransaction will create a new transaction draft
+// New transaction draft godoc
+// @Summary		New transaction draft
+// @Description	New transaction draft
+// @Tags		New Transactions
+// @Produce		json
+// @Param		NewTransaction body NewTransaction true "NewTransaction model containing the transaction config and metadata"
+// @Success		201 {object} models.DraftTransaction "Created transaction"
+// @Failure		400	"Bad request - Error while parsing NewTransaction from request body or xpub not found"
+// @Failure 	500	"Internal Server Error - Error while creating transaction"
+// @Router		/v1/transactions/drafts [post]
+// @Security	x-auth-xpub
+func (a *Action) newTransactionDraft(c *gin.Context) {
+	a.newTransaction(c)
```

Comment:
If we keep this method, then IMHO we should do that in the oposite "order". I mean, this method should have the whole logic, and old method should only call it.

Reply:
Agree. It will be easy to clean old stuff later.